electron.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
I took it from the [Quick Start guide](http://electron.atom.io/docs/latest/tutorial/quick-start/) of electron so i don't think it's necessary to montion